lib/kubernetes-deploy/runner_task.rb

```diff
@@ -0,0 +1,133 @@
+# frozen_string_literal: true
+require 'tempfile'
+
+require 'kubernetes-deploy/kubeclient_builder'
+require 'kubernetes-deploy/ui_helpers'
+require 'kubernetes-deploy/kubectl'
+
+module KubernetesDeploy
+  class RunnerTask
+    include KubernetesDeploy::KubeclientBuilder
+    include Kubectl
```

Comment:
Why `KubeclientBuilder` has to be references with the parent const and `Kubectl` doesn't?